locale/es.po

```diff
@@ -1884,147 +1897,149 @@ msgid ""
 "Incompatible .mpy file. Please update all .mpy files. See http://adafru.it/"
 "mpy-update for more info."
 msgstr ""
+"Archivo .mpy incompatible. Actualice todos los archivos .mpy. "
+"Consulte http://adafru.it/mpy-update para más información".
 
 #: py/persistentcode.c:326
 msgid "can only save bytecode"
-msgstr ""
+msgstr "solo puede almacenar bytecode"
 
 #: py/runtime.c:206
 msgid "name not defined"
 msgstr "name no definido"
 
 #: py/runtime.c:209
 msgid "name '%q' is not defined"
-msgstr ""
+msgstr "name '%q' no esta definido"
 
 #: py/runtime.c:304 py/runtime.c:611
 msgid "unsupported type for operator"
-msgstr ""
+msgstr "tipo de operador no soportado"
 
 #: py/runtime.c:307
 msgid "unsupported type for %q: '%s'"
-msgstr ""
+msgstr "tipo no soportado para %q: '%s'"
 
 #: py/runtime.c:614
 msgid "unsupported types for %q: '%s', '%s'"
-msgstr ""
+msgstr "tipos no soportados para %q: '%s', '%s'"
 
 #: py/runtime.c:881 py/runtime.c:888 py/runtime.c:945
 msgid "wrong number of values to unpack"
-msgstr ""
+msgstr "numero erroneo de valores a descomprimir"
 
 #: py/runtime.c:883 py/runtime.c:947
 #, c-format
 msgid "need more than %d values to unpack"
-msgstr ""
+msgstr "necesita más de %d valores para descomprimir"
 
 #: py/runtime.c:890
 #, c-format
 msgid "too many values to unpack (expected %d)"
-msgstr ""
+msgstr "demasiados valores para descomprimir (%d esperado)"
 
 #: py/runtime.c:984
 msgid "argument has wrong type"
-msgstr ""
+msgstr "el argumento tiene un tipo erroneo"
 
 #: py/runtime.c:986
 msgid "argument should be a '%q' not a '%q'"
-msgstr ""
+msgstr "argumento deberia ser un '%q' no un '%q'"
 
 #: py/runtime.c:1123 py/runtime.c:1197
 msgid "no such attribute"
-msgstr ""
+msgstr "no hay tal atributo"
 
 #: py/runtime.c:1128
 msgid "type object '%q' has no attribute '%q'"
-msgstr ""
+msgstr "objeto de tipo '%q' no tiene atributo '%q'"
 
 #: py/runtime.c:1132 py/runtime.c:1200
 msgid "'%s' object has no attribute '%q'"
-msgstr ""
+msgstr "objeto '%s' no tiene atributo '%q'"
 
 #: py/runtime.c:1238
 msgid "object not iterable"
-msgstr ""
+msgstr "objeto no es iterable"
```

Comment:
tal vez "objeto no iterable" y se sigue entendiendo bien, considerando el ahorro de memoria

Reply:
go it! :+1: